source/includes/_endpoint-dataset.md

```diff
@@ -777,12 +777,13 @@ The following rules apply for all formats:
 
 Some format-specific properties and options:
 
-Format    | Attribute           | Description                                                           | Example
---------- | --------------------| --------------------------------------------------------------------- | --------------------------
-csv       | use_category_ids    | instead of category names export the fields as their numeric ids      | {"use_category_ids": true}
-spss      | var_label_field     | Use the variable's name/description as SPSS variable label            | {"var_label_field": "name"}
-spss      | prefix_subvariables | When True, will only export subvariables and not group them as arrays | {"prefix_subvariables": true}
-all       | include_personal    | Will include the user's personal variables on the exported file       | {"include_personal": true}
+Format    | Attribute           | Description                                                                         | Default
+--------- | --------------------| ----------------------------------------------------------------------------------- | --------------------------
+csv       | use_category_ids    | Instead of category names export the fields as their numeric ids                    | {"use_category_ids": false}
+csv       | missing_values      | Which value to use for printing missing values; allowed "dot", "reason", "blank"    | {"missing_values": "reason}
```

Comment:
I disagree that "" is too falsey, particularly when this field has no boolean meaning. And I don't know what "forgot value" means--if I truly forgot to specify a value, this wouldn't be here at all. 

I naturally have R in mind when I suggest the direct string input. See the 'na' argument to https://stat.ethz.ch/R-manual/R-devel/library/utils/html/write.table.html, which is what writes CSVs in R.